<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SrarchIncom</class>
 <widget class="QWidget" name="SrarchIncom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="widget_customplot" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>1061</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_searchyear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询某年12个月</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_searchyearmouth">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询某年某月每天</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Year">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_YearYear">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_showlegend">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关闭图例</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_detail">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_mouth">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
